simpeg/potential_fields/magnetics/simulation.py

```diff
@@ -1665,562 +1664,531 @@ def _gtg_diagonal_without_building_g(self, weights):
 
 
 class Simulation3DDifferential(BaseMagneticPDESimulation):
-    """
-    Secondary field approach using differential equations!
+    r"""A secondary field simulation for magnetic data.
+
+    Parameters
+    ----------
+    mesh : discretize.base.BaseMesh
+    survey : magnetics.survey.Survey
+    mu : float, array_like
+        Magnetic Permeability Model (H/ m). Set this for forward
```

Comment:
Why not invert for `susceptibility` instead, and have mu_0 multiplied under the hood? This is generally the quantity of interest.